README.md

```diff
@@ -33,6 +33,24 @@ In this example, we use WIMSE with HTTP Message Signatures to explore the use ca
                              └──────────────────┘     
 ``` 
 
+A sequence diagram of the steps is provided:
+
+```mermaid
+sequenceDiagram
+    participant ID as Identity Server (minispire)
+    participant A as Caller (A)
+    participant B as Recipient (B)
+
+    ID ->> A: Issue WIT
+    Note right of A: Identity server issues<br/>a Workload Identity Token (WIT)<br/>bound to A’s public key (cnf)
```

Comment:
The "cnf" suggests the public key is somehow called/aliased "cnf". Let's either say "via cnf claim" or omit the parens altogether.